Makefile

```diff
@@ -467,3 +467,8 @@ pr:
 	make lint && \
 	make update-book-cli && \
 	make test
+
+check-features:
+	cargo hack check \
+		--package reth-codecs \
+		--feature-powerset
```

Comment:
this is a good starting point to track powerset support

Reply:
Yes! if add it to workspace, compilation time can become enormous